story/utils.py

```diff
@@ -41,23 +41,11 @@ def get_num_options(num):
         except ValueError:
             print("Error invalid choice. ")
 
-
 def player_died(text):
-
-    # reg_phrases = ["You[a-zA-Z ]* die.", "you[a-zA-Z ]* die.", "You[a-zA-Z ]* die ", "you[a-zA-Z ]* die ",]
-    #
-    # for phrase in reg_phrases:
-    #     reg_expr = re.compile(phrase)
-    #     matches = re.findall(reg_expr, text)
-    #     if len(matches) > 0:
-    #         return True
-
-    dead_phrases = ["you die", "You die", "you died", "you are dead", "You died", "You are dead", "You're dead",
-                    "you're dead", "you have died", "You have died", "you bleed out"]
-    for phrase in dead_phrases:
-        if phrase in text:
-            return True
-    return False
+    text = text.lower()
+    you_dead_regexps = ["you('re| are) (dead|killed)", "you die", "you('ve| have) (died|been killed)",
+                        "you \w+ to death"]
```

Comment:
I support this change because now I can fart myself to death!

Reply:
Hehe

Add `"you \w+ yourself to death"`

Reply:
Sure why not xD